.kitchen.yml

```diff
@@ -45,6 +45,14 @@ suites:
       require_chef_omnibus: 12.21.3
     run_list:
       - recipe[smoke::default]
+  - name: 3x-chef12
```

Comment:
Could you update matrix for test targets?

https://github.com/treasure-data/chef-td-agent/blob/8a561f2a0d99440fa8873079ca98aa62fad81da3/.travis.yml#L16-L24

Reply:
should we add one for 3x-chef13 too?